jwst/reset/reset_sub.py

```diff
@@ -35,11 +36,16 @@ def do_correction(input_model, reset_model):
 
     """
 
+    # Create output as a copy of the input science data model
+    output = input_model
```

Comment:
Again, I don't understand the purpose of copying, then closing.  Why not just continue to use `input_model`?

Reply:
thank you for catching this!